reddit_gold/templates/snoovatar.html

```diff
@@ -63,12 +63,15 @@
     %else:
       <div class="username">${thing.username}</div>
     %endif
-    <p>${_md('view and discuss snoovatars in /r/snoovatars')}</p>
+    <p>${_md('see more snooavatars and make suggestions in /r/snoovatars')}</p>
     <p>
       %if c.user.gold and c.user.name != thing.username:
         ${_md('create your own snoovatar [here](/user/%(username)s/snoo)' % dict(username=c.user.name))}
       %elif not c.user.gold:
-        ${_md('create your own snoovatar with [reddit gold](http://www.reddit.com/gold?source=snoovatar)')}
+        <div class="spacer"></div>
+        <div class="gold-accent rounded">
+          ${_md('[create your own snoovatar with reddit gold](/gold?source=snoovatar)')}
+        </div>
```

Comment:
It seems like you might want to wrap the gold-accent div in the spacer div.
`<div class="spacer">`
`  other div`
` </div>`